### ISSUE TITLE
redis-cli not responding when network is active

### ISSUE THREAD

**Issue by noilda** [docker-library/redis#252]

**Comment by noilda** [comment]
Please delete
Wrong place wrong issue.

**Comment by noilda** [comment]
Finally I fix it.
It is not a problem with redis. 
Something has changed to ubuntu or to docker and DNS was wrong.
Tried to put DNS on docker configurations file or to run containers by giving the DNS while running but no success .
Finally I did 
sudo systemctl stop docker
sudo iptables -t nat -F
sudo ifconfig docker0 down
sudo brctl delbr docker0
sudo systemctl start docker

Not sure exactly what those commands do but I can understand that they stop docker --> flush the rules of nat chain --> deactivate the docker0 interface --> delete docker bridge --> start docker
When starting docker will recreate again the rules on iptables and will create the interface and the bridge.
Also I deleted my custom networks and recreated them.It was not working without deleting and recreating my custom networks.
I believe it was not going to work even if I was attaching to default docker network. I tried from inside an ubuntu container .
A big help has been an answer on Stackoverflow by  Romeo Mihalcea (https://stackoverflow.com/a/20431030).
